app/assets/javascripts/admin/controllers/admin-web-hooks-show.js.es6

```diff
@@ -0,0 +1,32 @@
+import { popupAjaxError } from 'discourse/lib/ajax-error';
+import { default as computed, on, observe } from 'ember-addons/ember-computed-decorators';
+
+export default Ember.Controller.extend({
+  @computed('model.isSaving')
+  savingStatus(isSaving) {
+    return isSaving ? I18n.t('saving') : '';
+  },
+
+  wildcardEvent: Ember.computed('model.wildcard_event', {
```

Comment:
This can be written as https://github.com/rwjblue/ember-computed-decorators#real-world-getset-syntax using `ember-computed-decorators`